packages/pwa-kit-runtime/src/utils/ssr-paths.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export const startsWithMobify = (url) => {
+    return url.startsWith(`/mobify`)
+}
+
+export const getProxyPathBase = () => {
+    return '/mobify/proxy'
```

Comment:
These values seem fit to be a constants than funcs

Reply:
They are constants for now yes.

The intent here is to lay the groundwork so that in subsequent PRs, I can do something like:

```
export const getProxyPathBase = () => {
    const namespace = getNamespace()
    return namespace ? `/${namespace}/mobify/proxy` : '/mobify/proxy'
}
```

Reply:
I've added a placeholder for getNamespace so the functions now contain a conditional rather than just returning a constant.

Reply:
I realized I didn't need the conditional if namespace is blank so this now looks like:

```
export const getProxyPathBase = () => {
    const namespace = getNamespace()
    return `/${namespace}/mobify/proxy`
}
```